File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/billing/services/InvoiceService.java
package com.metaengine.demo.billing.services;

import com.metaengine.demo.billing.aggregates.Invoice;
import java.util.List;

/** Application service exposing invoice operations. */
public class InvoiceService {
    /** Create an invoice from the provided partial input. */
    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an invoice by id. Returns null when not found. */
    public Invoice findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} invoices. */
    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an invoice by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-invocation-heredoc/run-002/b-baseline/output/src/domain/billing/services/InvoiceService.ts
import { Invoice } from "../aggregates/Invoice";

/** Application service exposing invoice use cases. */
export class InvoiceService {
  create(input: Partial<Invoice>): Invoice {
    void input;
    throw new Error("not implemented");
  }

  findById(id: string): Invoice | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): Invoice[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}
